.gitea/workflows/test-ydb.yml

```diff
@@ -0,0 +1,61 @@
+name: test ydb
+on: 
+  push:
+    branches:
+      - master
+  pull_request:
+
+env:
+  GOPROXY: https://goproxy.io,direct
+  GOPATH: /go_path
+  GOCACHE: /go_cache
+
+jobs:
+  lint:
+    name: test ydb
+    runs-on: ubuntu-latest
+    steps:
+      # - name: cache go path
```

Comment:
Is this really need in upstream?

Reply:
no, but all CI files in upstream have that comment part, I'm not sure if in future XORM maintainer will use it or not